core/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -209,6 +223,7 @@ case class ScalafmtConfig(
     danglingParentheses: Boolean = false,
     poorMansTrailingCommasInConfigStyle: Boolean = false,
     bestEffortInDeeplyNestedCode: Boolean = false,
+    verticalMultiline: Boolean = false,
```

Comment:
Naming is hard. I would love to use an adt for styles supported in parameter lists with possible values bin pack, default and vertical multi-line. Currently it's a boolean in BinPack.

I'm OK with making this a boolean in scalafmt config, but the  name needs to be more specific. I suggest verticalMultilineAtDefinitionSite